vmslave-config/pipeline/vmslave_config.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+git diff --dirstat=files ${GIT_PREVIOUS_COMMIT} | grep "vmslave-config/"
```

Comment:
this line will compare if any change made in on-build-config/vmslave-config sub-folder....